apps/blur/halide_blur_generator.cpp

```diff
@@ -33,7 +33,7 @@ class HalideBlur : public Halide::Generator<HalideBlur> {
 
     Func build() {
         Func blur_x("blur_x"), blur_y("blur_y");
-        Var x("x"), y("y"), xi("xi"), yi("yi");
+        Var x("x"), y("y"), xi("xi"), yi("yi"), tx("tx"), ty("ty");
```

Comment:
Yeah, I am aware of that. At some point, I was using "tx and ty" in the test cases, but Andrew pointed out it is also used for tiling which made it confusing. The other gpu_tile usages in test also use xi/yi right now.